django_project/certification/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+
+from certificate import *
+from attendee import *
+from course import *
+from certifying_organisation import *
+from course_type import *
+from training_center import *
+from course_convener import *
```

Comment:
Or alternatively, you can have my settings or @ismailsunni :

Rename this to python-style.xml
[python-style.xml.txt](https://github.com/kartoza/projecta/files/866551/python-style.xml.txt)

Then import it:
![import](https://cloud.githubusercontent.com/assets/831865/24275827/e632ff98-1064-11e7-9b82-5d9fbe1d2b23.gif)